dom-querying/dom-query-quiz-notes.md

```diff
@@ -0,0 +1,58 @@
+# dom-query-quiz-notes
+
+## Quiz Questions
+
+Answer the following questions in the provided markdown file before turning in this exercise:
+
+- Why do we log things to the console?
+  To check our code for the expected results
+- What is a "model"?
+  A representation of all page content as objects that can be modified
+- Which "document" is being referred to in the phrase Document Object Model?
+  the main entry point to the page
+- What is the word "object" referring to in the phrase Document Object Model?
+  The whole document represented as an object that can be modified
+- What is a DOM Tree?
+  A representation of HTML as a tree structure of tags
+- Give two examples of `document` methods that retrieve a single element from the DOM.
+  getElementsById(), activeElement()
+- Give one example of a `document` method that retrieves multiple elements from the DOM at once.
+  cookie
```

Comment:
Not quite. Which method did you use in this exercise to select multiple elements?